POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/MainPage.xaml.cs

```diff
@@ -20,8 +20,19 @@ private async void powerOnLaunchDialog()
             onLaunchContentDialog dialog = new onLaunchContentDialog();
             dialog.PrimaryButtonClick += Dialog_PrimaryButtonClick;
             await dialog.ShowAsync();
+        }
+
+        /// <summary>
+        ///     This is currently called when the "Check for Updates" button is clicked (as a placeholder trigger for now)
+        ///     For the future: customize to calling only once after update is installed, etc.
+        /// </summary>
+        private async void DisplayUpdateDialog()
```

Comment:
Redundant description.  Unnecessary.  You might have
//TODO: Customize to call only once after update is installed, etc..

everything else is not adding anything to this method.

Reply:
Good point